tests/integration/test_helpers/checks_nondestructive.py

```diff
@@ -0,0 +1,26 @@
+import os
+import re
+
+import pytest
+
+from test_helpers.common_functions import get_log_file_data as log_file_data
+
+
+@pytest.fixture(autouse=True)
```

Comment:
You will need to add this file also here - https://github.com/oamg/convert2rhel/blob/main/tests/integration/conftest.py#L41, otherwise this "autosue=True" fixture will not run

Reply:
Thanks!